Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.cs

```diff
@@ -222,8 +222,13 @@ private void DetermineOneRowMode()
 
         private void RecalculateLayout(double containerWidth)
         {
+            var itemsWrapGridPanel = ItemsPanelRoot as ItemsWrapGrid;
```

Comment:
There is a check for null on the panel before using it so it could change. Best to use Panel